docusearch/management/commands/import_documents.py

```diff
@@ -67,10 +73,10 @@ def create_document(document, release_slug):
 
     d.release_agency_slug = release_slug
 
+    doc_file = File(open(doc_path, 'rb'))
     filename = os.path.basename(doc_path)
-    doc_path = 'docusearch/pdfs/%s' % filename
-    d.path = doc_path
-    d.save()
+
+    d.original_file.save(filename, doc_file, save=True)
```

Comment:
I'm getting a `BrokenPipeError: [Errno 32] Broken pipe`, even after I updated the AWS keys. 
```
# Uses s3boto
DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
AWS_ACCESS_KEY_ID = 'mykey'
AWS_SECRET_ACCESS_KEY = 'secret'
AWS_STORAGE_BUCKET_NAME = '18f-foia-docs'
```